### PR TITLE
cloud wallet example

### PR DESCRIPTION
**Pull request by maycon-mello**
This is a functional example of cloud wallet integration.
I want to perform some additional cleanup in another PR, plus few UX improvements.

### CODE REVIEW

File: web-example/src/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-import './sdk-test';
+// import './sdk-test';
```
Comment (gasher) [inline web-example/src/index.js:7]: nitpick: since the file was deleted this comment can also be deleted

File: web-example/craco.config.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
-const path = require('path');
-const webpack = require('webpack');
-const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
```
Comment (cykoder) [inline web-example/craco.config.js:3]: why the changes from `'` to `"`? across all other dock repos we use single quotes, best to keep consistent i think
